odd-platform-api/src/main/resources/application.yml

```diff
@@ -53,4 +53,4 @@ logging:
           interceptor: trace
       jooq:
         tools:
-          LoggerListener: debug
+          LoggerListener: info
```

Comment:
That's cool, thanks. It was spamming a lot.

Reply:
Agreed, but sometimes it's incredibly useful. Can be turned on manually however